### PR TITLE
connect frontend to ocr api in dev env

### PR DESCRIPTION
**Pull request by zdeveloper**
## Description
connect frontend to ocr api in dev env

## Related Issues
#269

### CODE REVIEW

File: frontend/.env.dev [hunk 1]
```diff
@@ -0,0 +1 @@
+VITE_API_URL=https://reportvision-ocr-api-dev.azurewebsites.net/
```
Comment (derekadombek) [inline frontend/.env.dev:1]: this should work but will probably just need to parametrized the environment tags eventually.

Comment (derekadombek) [reply frontend/.env.dev:1]: something like `https://reportvision-ocr-api-$ENV.azurewebsites.net/

Comment (zdeveloper) [reply frontend/.env.dev]: yeah seems like the .env files are not working, putting it directly into the build command works

File: .github/workflows/build-deploy-frontend.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Install NPM packages
         run: npm ci
       - name: Build project
-        run: npm run build 
+        run: VITE_API_URL=https://reportvision-ocr-api-dev.azurewebsites.net/ npm run build
```
Comment (zdeveloper) [inline .github/workflows/build-deploy-frontend.yml:28]: hard coding just for now to get dev working, will leave it to @derekadombek to parameterize

**Review by knguyenrise8 — APPROVED** [review]

**Pull request merged**